src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1664,7 +1664,6 @@ template<> void S6Importer::ImportEntity<Vehicle>(const RCT12SpriteBase& baseSrc
     dst->dodgems_collision_direction = src->dodgems_collision_direction;
     dst->animation_frame = src->animation_frame;
     dst->animationState = src->animationState;
-    dst->var_CA = src->var_CA;
```

Comment:
The import is not correct, it needs to combine var_CA and animationState into the single field now.